### PR TITLE
Updating Group-Object cmdlet

### PR DESCRIPTION
**Pull request by bobbytreed**
Version(s) of document impacted
------------------------------
- [ ] Impacts 6.next document
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[40e52a8](https://github.com/bobbytreed/PowerShell-Docs/pull/4228/commits/40e52a838e45978f06a3708cf1691b80274ecc57)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Utility/Group-Object.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-bobbytreed-b025/reference/3.0/Microsoft.PowerShell.Utility/Group-Object.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/group-object?view=powershell-3.0&branch=pr-en-us-4228) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Utility/Group-Object.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-bobbytreed-b025/reference/4.0/Microsoft.PowerShell.Utility/Group-Object.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/group-object?view=powershell-4.0&branch=pr-en-us-4228) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Utility/Group-Object.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-bobbytreed-b025/reference/5.0/Microsoft.PowerShell.Utility/Group-Object.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/group-object?view=powershell-5.0&branch=pr-en-us-4228) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Utility/Group-Object.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-bobbytreed-b025/reference/5.1/Microsoft.PowerShell.Utility/Group-Object.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/group-object?view=powershell-5.1&branch=pr-en-us-4228) (powershell-5.1) |
[reference/6/Microsoft.PowerShell.Utility/Group-Object.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-bobbytreed-b025/reference/6/Microsoft.PowerShell.Utility/Group-Object.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/group-object?view=powershell-6&branch=pr-en-us-4228) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C26%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201904261622080248-4228%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=o1raK%2FYYudbDmAu6C42EMhpL26LNU2Ig3a9y3GhmkMI%3D&st=2019-04-26T16%3A22%3A32Z&se=2019-05-27T16%3A27%3A32Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/3.0/Microsoft.PowerShell.Utility/Group-Object.md [hunk 1]
```diff
@@ -23,67 +22,102 @@ Group-Object [[-Property] <Object[]>] [-NoElement] [-AsHashTable] [-AsString]
...
+### Example 2: Group integers by odds and evens
 
-### Example 3
+This example shows how to use script blocks as the value of the **Property** parameter.
+
+This command displays the integers from 1 to 20, grouped by odds and even.
 
 ```powershell
-1..35 | Group-Object -Property {$_ % 2},{$_ % 3}
+1..20 | Group-Object -Property @{label="Odd",Expression{$_ % 2}}
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Utility/Group-Object.md]: This example does not work.
Try this:
```
1..20 | Group-Object -Property {$_ % 2}
```

File: reference/6/Microsoft.PowerShell.Utility/Group-Object.md [hunk 2]
```diff
@@ -22,66 +22,102 @@ Group-Object [-NoElement] [-AsHashTable] [-AsString] [-InputObject <PSObject>] [
...
-This command displays the integers from 1 to 35, grouped by the remainder left when they are divided by 2 or 3.
+These commands display the 1,000 most recent entries in the System event log, grouped by **EntryType**.
 
-### Example 4: Group event log events by ID
+In the output, the **Count** column represents the number of entries in each group, the **Name**
+column represents the **EventType** values that define a group, and the **Group** column represents
+the objects in each group.
 
+```powershell
+Get-EventLog -LogName "system" -Newest 1000 | Group-Object -Property EntryType
```
Comment (sdwheeler) [inline reference/6/Microsoft.PowerShell.Utility/Group-Object.md]: Get-EventLog does not exist in v6. So this example is not valid. Use this:

```powershell
Get-WinEvent -LogName System -MaxEvents 1000 | Group-Object -Property LevelDisplayName
```

```Output
Count Name                      Group
----- ----                      -----
  153 Error                     {System.Diagnostics.Eventing.Reader.EventLogRecord, System.Diagnostics.…
  722 Information               {System.Diagnostics.Eventing.Reader.EventLogRecord, System.Diagnostics.…
  125 Warning                   {System.Diagnostics.Eventing.Reader.EventLogRecord, System.Diagnostics.…
```

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[4fb15cc](https://github.com/bobbytreed/PowerShell-Docs/pull/4228/commits/4fb15cce2878f12f4b889c2449d50955c197b035)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Utility/Group-Object.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-bobbytreed-b025/reference/3.0/Microsoft.PowerShell.Utility/Group-Object.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/group-object?view=powershell-3.0&branch=pr-en-us-4228) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Utility/Group-Object.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-bobbytreed-b025/reference/4.0/Microsoft.PowerShell.Utility/Group-Object.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/group-object?view=powershell-4.0&branch=pr-en-us-4228) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Utility/Group-Object.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-bobbytreed-b025/reference/5.0/Microsoft.PowerShell.Utility/Group-Object.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/group-object?view=powershell-5.0&branch=pr-en-us-4228) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Utility/Group-Object.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-bobbytreed-b025/reference/5.1/Microsoft.PowerShell.Utility/Group-Object.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/group-object?view=powershell-5.1&branch=pr-en-us-4228) (powershell-5.1) |
[reference/6/Microsoft.PowerShell.Utility/Group-Object.md](https://github.com/bobbytreed/PowerShell-Docs/blob/release-bobbytreed-b025/reference/6/Microsoft.PowerShell.Utility/Group-Object.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/group-object?view=powershell-6&branch=pr-en-us-4228) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C26%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201904261741514811-4228%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=EPvhmHmfgIWzLqYksEcs2zlXuhGdIZ73VQyQ9RTW98I%3D&st=2019-04-26T17%3A41%3A39Z&se=2019-05-27T17%3A46%3A39Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**